pinot-plugins/pinot-stream-ingestion/pinot-kinesis/src/main/java/org/apache/pinot/plugin/stream/kinesis/KinesisConfig.java

```diff
@@ -48,6 +61,14 @@ public class KinesisConfig {
   private final String _secretKey;
   private final String _endpoint;
 
+  // IAM Role values
+  private boolean _iamRoleBasedAccess;
```

Comment:
can you add some javadoc on what these configs mean and which ones are optional ?